spec/new_inject_method_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'new_inject_method.rb'
+
+describe Array do
+
+  let (:test_array){[1,2,3,4]}
+
+  context 'addition' do
+    it 'can add numbers together' do
+      expect(test_array.christian_inject {|k,v| k + v}).to eq 10
+    end
+
+    it 'can add strings together' do
+      expect(['welcome','to','makers'].christian_inject {|k,v| k + v}).to eq "welcometomakers"
+    end
+
+    it 'can add numbers to an input' do
+      expect(test_array.christian_inject(10){|k,v| k + v}).to eq 20
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.